internal/clientgen/openapi/openapi.go

```diff
@@ -152,7 +153,7 @@ func (g *Generator) newOperationForEncoding(rpc *meta.RPC, method string, reqEnc
 				AllowEmptyValue: true,
 				AllowReserved:   false,
 				Deprecated:      false,
```

Comment:
Ohh nice, in the future we could even detect this and include it in the meta

Reply:
Yeah that's a good idea :)